scripts/anvil_tools/post_workspace_attributes.py

```diff
@@ -44,34 +43,60 @@ def setup_single_data_delivery_workspace(request, project="anvil-datastorage"):
         return workspace_dict
 
     workspace_dict["publish_workspace_status"] = "Success"
+    workspace_dict["final_workspace_status"] = "Success"
     return workspace_dict
 
 
-def setup_data_delivery_workspaces(tsv):
-    """Create array of json requests per row in tsv file."""
+def format_dataframe(workspace_row):
+    """Format workspace row Series object and convert to json."""
+
+    list_dtype_cols = ["library:consentCodes",
```

Comment:
will this list ever change? is it all `library:` keys or just this specific subset? just thinking about whether this could ever cause unexpected behavior

Reply:
its a specific subset that is defined in an attributes_definitions.json that Ruchi shared with me and she said it hasn't changed/probabaly won't change

Reply:
cool! maybe add a comment saying where the list came from and where to check for it in case of updates?